sunpy/map/map_factory.py

```diff
@@ -287,7 +287,7 @@ def __call__(self, *args, composite=False, sequence=False, silence_errors=False,
                     ValidationFunctionError, MapMetaValidationError) as e:
                 if not silence_errors:
                     raise
-                warnings.warn(f"One of the data, header pairs failed to validate with: {e}")
+                log.warning(f"One of the data, header pairs failed to validate with: {e}")
```

Comment:
we should still be `warnings.warn` here shouldn't we?

Reply:
Does that pass it to the logger?

Reply:
yes, the logger captures warnings.

Reply:
Fixed.